Readme.md

```diff
@@ -63,6 +63,8 @@ module "ecs" {
   cluster_cpu_scale_out = 70
   # scale cluster in on cpu %
   cluster_cpu_scale_in = 20
+  # list of instance metrics we'd like to enable
```

Comment:
since we are showing all the available options in the readme it might make sense to note that the default value in the module is an empty array, and the additional costs.

`# list of instance metrics we'd like to enable, default to empty array, additional collection may cause charges!`